src/Common/Collection.php

```diff
@@ -12,6 +12,8 @@ class Collection extends ArrayCollection
     private $count;
     /** @var int */
     private $offset;
+    /** @var int */
+    private $unreadNotLoaded;
```

Comment:
I don't think we would need this property for user collection for example. It should not be here. You can create a new collection to extend if you need it for channels, groups, ims, or move it to ima collection if it's needed there only.